docs/specs/vorto_repository-1.0.yml

```diff
@@ -700,9 +809,42 @@ paths:
         404:
           description: The request could not be completed, resource not found.
       deprecated: false
-      
+
 components:
   schemas:
+    Collaborator:
+      title: Collaborator
+      type: object
+      properties:
+        userId: 
+          description: UserId of collaborator
+          type: string
+        providerId:
+          description: The provider ID (e.g Bosch, Github) of the user
```

Comment:
Possible values are 
* Github
* BoschID
* BoschSuiteAuth

The value is validated against the Verification Provider